CMakeLists.txt

```diff
@@ -86,7 +86,7 @@ set(CMAKE_MODULE_PATH
 
 set( CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin )
 set( CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib )
-
+set (CMAKE_SKIP_INSTALL_ALL_DEPENDENCY TRUE)
```

Comment:
I will leave a comment to the relevant changes, as the majority is just due to the license replacement.